### DISCUSSION TITLE
access to pyrep repo

### DISCUSSION THREAD

**Discussion by davel94** [ansys/pyansys-dev-guide#181] [General]
Hi,

I would like access to the pyrep repo to submit a PR.  Adding to example scripts.

thanks
Dave

**Comment by MaxJPRey** [comment]
Pinging @jonathanzopes  and @nezgrath who are contributors of the pyrep repo.

**Reply by jonathanzopes** [reply]
Done.

**Reply by MaxJPRey** [reply]
Thanks @jonathanzopes !
